pyproject.toml

```diff
@@ -51,9 +51,6 @@ ignore = [
     "UP032",
 ]
 
-[tool.setuptools_scm]
-version_file = "vllm/_version.py"
-
```

Comment:
This is the proper way to configure `setuptools_scm` but we do not have a `[project]`, which is required for `setuptools-scm` in order to set `dynamic = [ "version" ]`. This will do for now, but we'lll have to migrate project definitions from `setup.py` to `pyproject.toml` at some point.

Reply:
Here's a PR for that https://github.com/vllm-project/vllm/pull/8772